pyxem/generators/indexation_generator.py

```diff
@@ -52,108 +50,25 @@ class IndexationGenerator:
 
     Returns
     -------
-    ValueError : "use TemplateIndexationGenerator or VectorIndexationGenerator"
+    ValueError : "use AcceleratedIndexationGenerator or VectorIndexationGenerator"
     """
 
     def __init__(self, signal, diffraction_library):
-        raise ValueError("use TemplateIndexationGenerator or VectorIndexationGenerator")
+        raise ValueError("use AcceleratedIndexationGenerator or VectorIndexationGenerator")
 
-
-def _correlate_templates(image, library, n_largest, method, mask):
-    r"""Correlates all simulated diffraction templates in a DiffractionLibrary
-    with a particular experimental diffraction pattern (image).
-
-    Calculated using the normalised (see return type documentation) dot
-    product, or cosine distance,
-
-    .. math:: fast_correlation
-        \\frac{\\sum_{j=1}^m P(x_j, y_j) T(x_j, y_j)}{\\sqrt{\\sum_{j=1}^m T^2(x_j, y_j)}}
-
-    .. math:: zero_mean_normalized_correlation
-        \\frac{\\sum_{j=1}^m P(x_j, y_j) T(x_j, y_j)- avg(P)avg(T)}{\\sqrt{\\sum_{j=1}^m (T(x_j, y_j)-avg(T))^2+\sum_{j=1}^m P(x_j,y_j)-avg(P)}}
-        for a template T and an experimental pattern P.
+class TemplateIndexationGenerator:
```

Comment:
The changelog says this class is removed, but it's still here. I assume this is to make unaware users switch to the new class. When will it be removed? A TODO comment stating when to remove it, with a reference to the removal in an issue so we don't forget it should do I think.

Reply:
I agree, will make the adjustments.